templates/extension-template-ts/src/index.ts

```diff
@@ -1,4 +1,6 @@
-import { JsPsych, JsPsychExtension, JsPsychExtensionInfo } from "jspsych";
+import { JsPsych, JsPsychExtension, JsPsychExtensionInfo, ParameterType } from "jspsych";
+
+import { version } from "../package.json";
```

Comment:
is this the right path??

Reply:
For the .ts files which are in a /src folder it is the right path.